src/ORM/Rule/ExistsIn.php

```diff
@@ -40,14 +40,28 @@ class ExistsIn
     protected $_repository;
 
     /**
+     * Options for the constructor
+     *
+     * @var array
+     */
+    protected $_options = [];
```

Comment:
@markstory We haven't suppressed any warnings when running phpcs on travis https://github.com/cakephp/cakephp/blob/master/.travis.yml#L77   All suppressions are done through `CakPHP` rulesset itself.